src/alchemlyb/visualisation/mbar_martix.py

```diff
@@ -0,0 +1,81 @@
+"""Functions for Plotting the overlay matrix for the Mbar estimator.
+
+"""
+from __future__ import division
+import matplotlib
+matplotlib.use('Agg')
```

Comment:
@orbeckst Thank you for the review. I got a problem with CI for py2, where if I don't add this line in. I would get 
```
src/alchemlyb/tests/test_visualisation.py:15: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
src/alchemlyb/visualisation/mbar_martix.py:31: in plot_mbar_omatrix
    fig, ax = plt.subplots(figsize=(size / 2, size / 2))
../../../virtualenv/python2.7.15/lib/python2.7/site-packages/matplotlib/pyplot.py:1184: in subplots
    fig = figure(**fig_kw)
../../../virtualenv/python2.7.15/lib/python2.7/site-packages/matplotlib/pyplot.py:533: in figure
    **kwargs)
../../../virtualenv/python2.7.15/lib/python2.7/site-packages/matplotlib/backend_bases.py:161: in new_figure_manager
    return cls.new_figure_manager_given_figure(num, fig)
../../../virtualenv/python2.7.15/lib/python2.7/site-packages/matplotlib/backends/_backend_tk.py:1046: in new_figure_manager_given_figure
    window = Tk.Tk(className="matplotlib")
```
As is seen https://travis-ci.org/github/alchemistry/alchemlyb/jobs/720700286, Google suggests that this is one way of solving the problem. I'm not familiar with CI so I'm not sure of what is the best way to process.

Reply:
@orbeckst I'm sorry for the confusion. I originally put the line `matplotlib.use('Agg')` in order to avoid the travis failing for py2. I wonder what is your suggestion?

Reply:
Sorry to be brief: see https://github.com/MDAnalysis/mdanalysis/pull/2798/files#diff-354f30a63fb0907d4ad57269548329e3R39 : add the env var `MPLBACKEND: "agg"` to the CI.